mlxtend/association/association_rules.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pandas as pd
+from itertools import combinations
+
+def association_rules(df, metric="confidence", min_threshold=0.8):
+    """"""
+    # Metrics for association rules
+    metric_dict = {
+        "confidence": lambda sXY, sX, _: \
+        sXY/sX,
+        "lift": lambda sXY, sX, sY:\
+        metric_dict["confidence"](sXY, sX, sY)/sY
+        }
+    
+    # check for metric compliance
+    if metric not in metric_dict.keys():
+        raise ValueError("Metric must be 'confidence' or 'lift', got '{}'".format(metric))
+        
+    # get dict of {frequent itemset} -> support
+    frozenset_vect = np.vectorize(lambda x: frozenset(x))
+    keys = df.values.T[1]
```

Comment:
I think using 

    keys = df['itemsets'].values
    values = df['support'].values

could be a safer approach. It would throw an error if the exact columns are not present in the input dataset rather than doing something unexpected where the code may run but produces weird results (e.g,. if someone swapped the column order or so)